ci-operator/config/shiftstack/merge-bot/shiftstack-merge-bot-main__periodic-4.12.yaml

```diff
@@ -0,0 +1,119 @@
+      USER 0:0
+      WORKDIR /go
+      RUN chown 1001:1001 .
+
+      USER 1001:1001
+images:
+- dockerfile_path: Containerfile
+  to: merge-bot
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+test_binary_build_commands: make install-all-deps
+tests:
+- as: lint
+  commands: make lint
+  container:
+    from: test-bin
+- as: unittests
+  commands: make unittests
+  container:
+    from: test-bin
+- as: cloud-provider-openstack
+  cron: 0 12 * * Mon,Thu
+  steps:
+    test:
+    - as: cloud-provider-openstack
+      commands: |
+        merge-bot --source https://github.com/kubernetes/cloud-provider-openstack:release-1.25 \
+                  --dest openshift/cloud-provider-openstack:release-4.12 \
+                  --merge shiftstack/cloud-provider-openstack:merge-bot-release-4.12 \
+                  --update-go-modules \
+                  --bot-name shiftstack-merge-bot --bot-email shiftstack-merge-bot@redhat.com \
+                  --github-app-key /secrets/merge-bot/github_private_key \
+                  --github-cloner-key /secrets/merge-bot/github_cloner_private_key \
+                  --slack-webhook /secrets/slack-hooks/forum-shiftstack
+      credentials:
+      - mount_path: /secrets/merge-bot
+        name: shiftstack-merge-bot
+        namespace: test-credentials
+      - mount_path: /secrets/slack-hooks
+        name: shiftstack-slack-hooks
+        namespace: test-credentials
+      from: merge-bot
+      resources:
+        limits:
+          memory: 4Gi
+        requests:
+          cpu: 100m
+          memory: 200Mi
+- as: csi-driver-nfs
+  cron: 0 12 * * Mon,Thu
+  steps:
+    test:
+    - as: csi-driver-nfs
+      commands: |
+        merge-bot --source https://github.com/kubernetes-csi/csi-driver-nfs:release-1.25 \
+                  --dest openshift/csi-driver-nfs:release-4.12 \
+                  --merge shiftstack/csi-driver-nfs:merge-bot-release-4.12 \
+                  --update-go-modules \
+                  --bot-name shiftstack-merge-bot --bot-email shiftstack-merge-bot@redhat.com \
+                  --github-app-key /secrets/merge-bot/github_private_key \
+                  --github-cloner-key /secrets/merge-bot/github_cloner_private_key \
+                  --slack-webhook /secrets/slack-hooks/forum-shiftstack
+      credentials:
+      - mount_path: /secrets/merge-bot
+        name: shiftstack-merge-bot
+        namespace: test-credentials
+      - mount_path: /secrets/slack-hooks
+        name: shiftstack-slack-hooks
+        namespace: test-credentials
+      from: merge-bot
+      resources:
+        limits:
+          memory: 4Gi
+        requests:
+          cpu: 100m
+          memory: 200Mi
+- as: kuryr-kubernetes
+  cron: 0 9 * * Mon,Wed,Fri
+  steps:
+    test:
+    - as: kuryr-kubernetes
+      commands: |
+        merge-bot --source https://opendev.org/openstack/kuryr-kubernetes:release-1.25 \
```

Comment:
kuryr-kubernetes is based on openstack releases and release-1.25 branch doesn't exist. I'm not sure how the kuryr project maintains stable branch but looks to me that we should remove kuryr from the stable branches sync automation.